systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -465,10 +491,57 @@ synchronized void createBigAmountOfUsers(ExtensionContext extensionContext, Stri
             LOGGER.info("Checking status of KafkaUser {}", userNameWithSuffix);
             Condition kafkaCondition = KafkaUserResource.kafkaUserClient().inNamespace(NAMESPACE).withName(userNameWithSuffix).get()
                     .getStatus().getConditions().get(0);
-            LOGGER.info("KafkaUser condition status: {}", kafkaCondition.getStatus());
-            LOGGER.info("KafkaUser condition type: {}", kafkaCondition.getType());
+            LOGGER.debug("KafkaUser condition status: {}", kafkaCondition.getStatus());
+            LOGGER.debug("KafkaUser condition type: {}", kafkaCondition.getType());
+            assertThat(kafkaCondition.getType(), is(Ready.toString()));
+            LOGGER.debug("KafkaUser {} is in desired state: {}", userNameWithSuffix, kafkaCondition.getType());
+        }
+    }
+
+    synchronized void alterBigAmountOfUsers(ExtensionContext extensionContext, String userName, String typeOfUser, int numberOfUsers,
+                                            int producerRate, int consumerRate, int requestsPercentage, double mutationRate) {
+        KafkaUserQuotas kuq = new KafkaUserQuotas();
+        kuq.setConsumerByteRate(consumerRate);
+        kuq.setProducerByteRate(producerRate);
+        kuq.setRequestPercentage(requestsPercentage);
+        kuq.setControllerMutationRate(mutationRate);
+
+        for (int i = 0; i < numberOfUsers; i++) {
+            String userNameWithSuffix = userName + "-" + i;
+            if (typeOfUser.equals("TLS")) {
+                resourceManager.createResource(extensionContext, KafkaUserTemplates.tlsUser(userClusterName, userNameWithSuffix)
+                    .editMetadata()
+                        .withNamespace(NAMESPACE)
+                    .endMetadata()
+                        .editSpec()
+                            .withQuotas(kuq)
+                        .endSpec()
+                    .build());
+            } else {
+                resourceManager.createResource(extensionContext, KafkaUserTemplates.scramShaUser(userClusterName, userNameWithSuffix)
+                    .editMetadata()
+                        .withNamespace(NAMESPACE)
+                    .endMetadata()
+                        .editSpec()
+                            .withQuotas(kuq)
+                        .endSpec()
+                    .build());
+            }
+
+            LOGGER.info("[After update] Checking status of KafkaUser {}", userNameWithSuffix);
```

Comment:
I wonder if we really need this useless log on INFO level when details are on DEBUG level?

Reply:
You can wonder the same for few lines above - with user creation - same story.
Let me know what you figured out. ;-)

Reply:
Move ti to DEBUG level and just log info before loops